src/lib/appManagers/appStateManager.ts

```diff
@@ -6,20 +6,19 @@
 
 import type { Dialog } from './appMessagesManager';
 import type { UserAuth } from '../mtproto/mtproto_config';
-import type { AppUsersManager } from './appUsersManager';
-import type { AppChatsManager } from './appChatsManager';
+import type { User } from './appUsersManager';
 import type { AuthState } from '../../types';
 import type FiltersStorage from '../storages/filters';
 import type DialogsStorage from '../storages/dialogs';
-import type { AppDraftsManager } from './appDraftsManager';
 import EventListenerBase from '../../helpers/eventListenerBase';
 import rootScope from '../rootScope';
 import sessionStorage from '../sessionStorage';
 import { logger } from '../logger';
 import { copy, setDeepProperty, validateInitObject } from '../../helpers/object';
-import { getHeavyAnimationPromise } from '../../hooks/useHeavyAnimationCheck';
 import App from '../../config/app';
 import DEBUG, { MOUNT_CLASS_TO } from '../../config/debug';
+import AppStorage from '../storage';
+import { Chat } from '../../layer';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MikeOwino/tweb/pullRequest?prid=7329411)
```suggestion
import { Chat } from "../../layer";
```